### ISSUE TITLE
Current issues

### ISSUE THREAD

**Issue by joseffaghihi** [joseffaghihi/GamificationAlgebraConcepts#26]
Current issues:

1) in the more difficult coefficient problems (x^2 = bx -5),

```
         a) The answers appear early. So, if I do not click at all I get two rewarding messages.
         b) When you miss three time the right answer the tutorial page appears BLACK and the minion still spawned
        c) there are some problem with the tutorial that I will talk to David.
```

**Comment by ghost** [comment]
I'll need to take a look at the second issue, that's for sure a bug if I can reproduce it.

**Comment by ghost** [comment]
I can confirm that the video doesn't play correctly; I'll investigate this during the coming weekend.

**Comment by ghost** [comment]
The second issue should be fixed, as I resolved that issue maybe a month ago.  The first issue merely ties into the AI.  Currently, the minions will retrieve the numbers just like before, but there is embedded controls to manage the entire numbering system which addresses your first complaint.  But is only triggered once a class calls it, which right now nothing does call it, as far as I remember.

**Comment by ghost** [comment]
Resolved the cheating issue in this commit [ 7b3ae6aae2ca4575b46645dc876f282edcb42455 ].  I'll go ahead and close this, please make a new ticket if there's any other issues.
